<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>832</width>
        <height>978</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="baseStationsGroup">
         <property name="title">
          <string>Base Stations</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BaseStationsWidget" name="baseStationsWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="usersGroup">
         <property name="title">
          <string>Users</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="UsersWidget" name="usersWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="reportsGroup">
         <property name="title">
          <string>Reports</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ReportsWidget" name="reportsWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="emailGroup">
            <property name="title">
             <string>Email</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="emailWidget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QWidget" name="widget" native="true">
                  <layout class="QFormLayout" name="formLayout">
                   <property name="verticalSpacing">
                    <number>12</number>
                   </property>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>SMTP server address</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="host">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="placeholderText">
                      <string>smtp.gmail.com</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>SMTP SSL Port</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QSpinBox" name="port">
                     <property name="maximum">
                      <number>65535</number>
                     </property>
                     <property name="value">
                      <number>465</number>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="0">
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Username</string>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="1">
                    <widget class="QLineEdit" name="username">
                     <property name="placeholderText">
                      <string>you@company.com</string>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Password</string>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="1">
                    <widget class="QLineEdit" name="password">
                     <property name="echoMode">
                      <enum>QLineEdit::PasswordEchoOnEdit</enum>
                     </property>
                    </widget>
                   </item>
                   <item row="7" column="0">
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>From field</string>
                     </property>
                    </widget>
                   </item>
                   <item row="7" column="1">
                    <widget class="QLineEdit" name="from">
                     <property name="text">
                      <string>Sense Monitoring System</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QWidget" name="widget_4" native="true">
                  <layout class="QFormLayout" name="formLayout_3">
                   <item row="0" column="0">
                    <widget class="QPushButton" name="sendTestEmail">
                     <property name="text">
                      <string>Send Test Email</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="testEmailTo">
                     <property name="placeholderText">
                      <string>Test Recipient</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="ftpGroup">
            <property name="title">
             <string>FTP</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="ftpWidget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QWidget" name="widget_2" native="true">
                  <layout class="QFormLayout" name="formLayout_2">
                   <property name="verticalSpacing">
                    <number>12</number>
                   </property>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>Server address</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="host_2">
                     <property name="text">
                      <string>smtp.gmail.com</string>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="label_12">
                     <property name="text">
                      <string>Username</string>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="1">
                    <widget class="QLineEdit" name="username_2"/>
                   </item>
                   <item row="5" column="0">
                    <widget class="QLabel" name="label_13">
                     <property name="text">
                      <string>Password</string>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="1">
                    <widget class="QLineEdit" name="password_2">
                     <property name="echoMode">
                      <enum>QLineEdit::PasswordEchoOnEdit</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QWidget" name="widget_5" native="true">
                  <layout class="QFormLayout" name="formLayout_4">
                   <item row="0" column="0" colspan="2">
                    <widget class="QPushButton" name="pushButton">
                     <property name="text">
                      <string>Test</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseStationsWidget</class>
   <extends>QWidget</extends>
   <header>BaseStationsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReportsWidget</class>
   <extends>QWidget</extends>
   <header>ReportsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UsersWidget</class>
   <extends>QWidget</extends>
   <header>UsersWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
